lib/routes/account.js

```diff
@@ -659,6 +659,14 @@ module.exports = function (
               log.event('verified', { email: account.email, uid: account.uid, locale: account.locale })
               log.increment('account.verified')
               return db.verifyEmail(account)
+                .then(mailer.sendPostVerifyEmail.bind(
+                    mailer,
+                    account.email,
+                    {
+                      acceptLanguage: request.app.acceptLanguage
+                    }
+                  )
+                )
```

Comment:
I'm *meh* on this. We didn't say they connected their device to "Sync", just to their "Firefox Account" *whatever __that__ means*, so its still technically correct despite being a practically useless statement.

I think its a good opportunity to promote Sync either way but the wording could be better.